app/src/main/java/com/helio/app/model/Day.java

```diff
@@ -0,0 +1,33 @@
+package com.helio.app.model;
+
+/**
+ * Represents the day of the week. DayOfWeek already exists in Java, but only works in Android API level 26+
+ */
+public enum Day {
+
+    MONDAY("Monday"),
+    TUESDAY("Tuesday"),
+    WEDNESDAY("Wednesday"),
+    THURSDAY("Thursday"),
+    FRIDAY("Friday"),
+    SATURDAY("Saturday"),
+    SUNDAY("Sunday");
+
+    public final String dayName;
+
+    Day(String dayName) {
+        this.dayName = dayName;
+    }
+
+    /**
+     * Gets a day enum from its string representation
+     */
+    public static Day getEnumFromName(String name) throws IllegalArgumentException {
+        for (Day day : Day.values()) {
```

Comment:
You could  consider[ the following method](https://stackoverflow.com/a/1080912) instead of a linear scan.
I don't it matters though, given that there are only 7 values.